testproject/Assets/Tests/Runtime/ObjectParenting/ParentinAutoSyncManager.cs

```diff
@@ -0,0 +1,64 @@
+using System.Collections.Generic;
+using UnityEngine;
+using Unity.Netcode;
+
+
+namespace TestProject.RuntimeTests
+{
+    /// <summary>
+    /// Helper class that builds 4 transform lists based on parent-child hierarchy
+    /// for the <see cref="ParentingInSceneObjectsTests.InSceneNestedAutoSyncObjectTest"/>
+    /// </summary>
+    public class ParentinAutoSyncManager : NetworkBehaviour
```

Comment:
```suggestion
    public class ParentingAutoSyncManager : NetworkBehaviour
```

Reply:
>.<  (DOH!)

Thank you for catching that spelling issue.
_Added the "g" to the class and filename._